index.html

```diff
@@ -64,5 +64,211 @@ <h3>LET'S CONNECT</h3>
+              </div>
+            </div>
+            <div class="see-work">
+              <a class="multi-edit" href="#">See Project</a>
+            </div>
+          </div>
+      </div>
+
+      <div class="works-wrapper">
+        <div class="works-section">
+            <div class="work-snapshot">
+              <img src="assets/works-snapshot/Snapshoot Portfolio 3.png" alt="work-snapshot-1">
+            </div>
+            <div class="work-description">
+              <div class="desc-wrapper">
+                <h2>Tonic</h2>
+                <div class="dec-details">
+                  <h4>CANOPY</h4>
+                  <img class="mobile-dot" src="assets/mobile/dot.png" alt="dot">
+                  <h4 class="grey-text">Back End Dev</h4>
+                  <img class="mobile-dot" src="assets/mobile/dot.png" alt="dot">
+                  <h4 class="grey-text">2015</h4>
+                </div>
+                  <p class="desc-text">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+                  <div class="work-tech">
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                  </div>
+              </div>
+            </div>
+            <div class="see-work">
+              <a href="#">See Project</a>
+            </div>
+          </div>
+      </div>
+       
+      <div class="works-wrapper">
+        <div class="works-section">
+        <div class="work-snapshot">
+            <img src="assets/works-snapshot/Snapshoot Portfolio 4.png" alt="work-snapshot-1">
+          </div>
+          <div class="work-description">
+            <div class="desc-wrapper">
+              <h2>Multi-Post<br>Stories</h2>
+              <div class="dec-details">
+                <h4>CANOPY</h4>
+                <img class="mobile-dot" src="assets/mobile/dot.png" alt="dot">
+                <h4 class="grey-text">Back End Dev</h4>
+                <img class="mobile-dot" src="assets/mobile/dot.png" alt="dot">
+                <h4 class="grey-text">2015</h4>
+              </div>
+              <p class="desc-text">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+              <div class="work-tech">
+                <ul>
+                  <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+              <div class="see-work">
+            <a class="multi-edit" href="#">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+  
+  <!-- ABOUT Section begins HERE------------------------------------------------------------ -->
+
+  <section class="headline about">
+    <div class="intro-phrase">
+          <h1>About<br>Myself</h1>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` for The title (e.g., "About myself") instead of `h1` tag ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.
- NB: There should be just one h1 in an HTML file, for SEO(Search Engine Optimization) and web accessibility purposes.